src/plugin/kubearmor.go

```diff
@@ -49,14 +49,17 @@ func ConvertKnoxSystemPolicyToKubeArmorPolicy(knoxPolicies []types.KnoxSystemPol
 		kubePolicy := types.KubeArmorPolicy{
 			APIVersion: "security.kubearmor.com/v1",
 			Kind:       "KubeArmorPolicy",
-			Metadata:   map[string]string{},
 		}
 
 		if policy.Kind != types.KindKubeArmorHostPolicy {
-			kubePolicy.Metadata["namespace"] = policy.Metadata["namespace"]
+			kubePolicy.Metadata.Namespace = policy.Metadata["namespace"]
 		}
 
-		kubePolicy.Metadata["name"] = policy.Metadata["name"]
+		kubePolicy.Metadata.Name = policy.Metadata["name"]
+		kubePolicy.Metadata.Annotations = map[string]string{
+			"app.accuknox.com/tldr":        policy.Metadata["annotation-tldr"],
+			"app.accuknox.com/description": policy.Metadata["annotation-description"],
+		}
```

Comment:
can we check to see if either of this fields  (' annotation-tldr` or `annotation-description`) are empty and then add an annotation 

```suggestion
		if policy.Metadata["annotation-tldr"] != "" || policy.Metadata["annotation-description"] != "" {
			kubePolicy.Metadata.Annotations = map[string]string{
				"app.accuknox.com/tldr":        policy.Metadata["annotation-tldr"],
				"app.accuknox.com/description": policy.Metadata["annotation-description"],
			}
		}
```

Reply:
addressed